<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glDockWidget</class>
 <widget class="QWidget" name="glDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QDockWidget" name="dw_example">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>800</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>示例栏</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="example_page">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tool_example">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>164</width>
                <height>636</height>
               </rect>
              </property>
              <attribute name="label">
               <string>示例</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tool_control">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>164</width>
                <height>636</height>
               </rect>
              </property>
              <attribute name="label">
               <string>示例参数</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2"/>
             </widget>
             <widget class="QWidget" name="tool_tips">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>164</width>
                <height>636</height>
               </rect>
              </property>
              <attribute name="label">
               <string>示例备注</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTextEdit" name="textEdit"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="detail_page"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
